src/sweep-line.js

```diff
@@ -1,4 +1,4 @@
-const SplayTree = require('splaytree')
+import SplayTree from 'splaytree'
```

Comment:
Kinda confused by this change... assuming it's intentional - do you know why splaytree needs to be imported like this and the other modules are ok with the `require()` ?

Reply:
Yeah I need to look into that further, it's probably a webpack config thing which I should be able to fix. ES6 modules and bundling tools are still a bit complicated sometimes.

Reply:
I think what's going on is that webpack is loading the es module for `splaytree` because it sees the [`module` definition in splaytree's `package.json`](https://github.com/w8r/splay-tree/blob/v0.1.1/package.json#L9) This is conflicting with the `require('splaytree')` call - which expects a common JS module.

I don't think this is the 'right' solution, but another way to avoid this conflict for the purposes of this PR is to make a change to the `webpack.config.js` to force the splaytree import to load the common JS version of splaytree rather than the es6 module. This is one way to get webpack to do that (maybe not the best way?):

```diff
--- a/webpack.config.js
+++ b/webpack.config.js
@@ -42,6 +42,7 @@ module.exports = {
   },
   resolve: {
     alias: {
+      'splaytree$': 'splaytree/dist/splay.js',
       'vue$': 'vue/dist/vue.esm.js'
     },
     extensions: ['*', '.js', '.vue', '.json']
```

Applying this change to `webpack.config.js` then makes this change from `require()` to `import` in `src/sweep-line.js` unnecessary, which I like because then these source files can all still be loaded directly by node with no intermediate compile/build step. Down the road, I would like to add `npm run build` step to build both common JS and es modules, but I think better to address that separately and not mixed in with this PR.

Thoughts?

Reply:
I'll try and take a look into this tonight, been away for the weekend sorry